packages/application-extension/src/index.tsx

```diff
@@ -316,6 +318,15 @@ const mainCommands: JupyterFrontEndPlugin<void> = {
         isVisible: () => true
       });
 
+      commands.addCommand(CommandIDs.toggleFullscreenMode, {
+        label: () => trans.__('Fullscreen Mode'),
+        execute: () => {
+          labShell.fullscreenMode = !labShell.fullscreenMode;
+        },
+        isToggled: () => labShell.fullscreenMode,
+        isVisible: () => true
```

Comment:
If attributes are constant you can set the value directly. They can be constant or callback.

I removed the `isVisible` as it is `true` by default.

```suggestion
        label: trans.__('Fullscreen Mode'),
        execute: () => {
          labShell.fullscreenMode = !labShell.fullscreenMode;
        },
        isToggled: () => labShell.fullscreenMode
```